tests/test_camera.py

```diff
@@ -99,14 +99,16 @@ def test_write(self):
         input_video.release()
         self.videowriter.release()
         output_video = cv2.VideoCapture(self.output_video_path)
+        # Open input video again to compare frames
+        input_video = cv2.VideoCapture(VIDEO_PATH)
 
         # Check whether new output file is the same as the original
         while True:
             ret, frame_in = input_video.read()
             _, frame_out = output_video.read()
             if not ret:
                 break
-            assert np.ndarray([frame_in == frame_out]).all()
+            assert (np.isclose(frame_in, frame_out, atol=50)).all()
```

Comment:
We could also use [`np.allclose`](https://numpy.org/doc/stable/reference/generated/numpy.allclose.html#numpy.allclose)

Reply:
Das is richtig :facepalm: